data_processing/integrity.py

```diff
@@ -0,0 +1,73 @@
+import pandas as pd
+
+# Checks the len of the dataframe
+def check_len(df, min):
+    if len(df) == 0:
+        raise Exception("Empty dataframe")
+    if len(df) < min:
+        raise Exception("Dataframe too short")
```

Comment:
its how you use it that matters

Reply:
vi har haft nogle problemer med at få filer fra test runs, som kun har et row